packages/react-components/react-tags-preview/src/components/InteractionTag/useInteractionTagStyles.styles.ts

```diff
@@ -99,6 +99,7 @@ const useContentButtonStyles = makeStyles({
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28686&commitId=d85b328e71c3ea4d58cd292fc78678c2a81cd3a5&env=prod&clientType=FLUENTUI)


  <details><summary><b>InteractionTag Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand - Dark Mode.pressed content.chromium.png|144| Changed|
|InteractionTag Converged.Brand - High Contrast.hover content.chromium.png|91| Changed|
|InteractionTag Converged.Brand - Dark Mode.default.chromium.png|16| Changed|
|InteractionTag Converged.Brand - Dark Mode.hover dismiss.chromium.png|17| Changed|
|InteractionTag Converged.Brand - Dark Mode.pressed dismiss.chromium.png|17| Changed|
|InteractionTag Converged.Brand.default.chromium.png|9| Changed|
|InteractionTag Converged.Brand - Dark Mode.hover content.chromium.png|36| Changed|
|InteractionTag Converged.Brand - High Contrast.pressed dismiss.chromium.png|9| Changed|
|InteractionTag Converged.Brand.hover content.chromium.png|5| Changed|
|InteractionTag Converged.Brand - High Contrast.pressed content.chromium.png|91| Changed|
|InteractionTag Converged.Brand - High Contrast.hover dismiss.chromium.png|9| Changed|
|InteractionTag Converged.Brand.pressed dismiss.chromium.png|9| Changed|
|InteractionTag Converged.Brand.pressed content.chromium.png|35| Changed|
|InteractionTag Converged.Brand.hover dismiss.chromium.png|9| Changed|
|InteractionTag Converged.Outline.hover content.chromium.png|37| Changed|
|InteractionTag Converged.Outline - Dark Mode.hover content.chromium.png|78| Changed|
|InteractionTag Converged.Outline - Dark Mode.pressed content.chromium.png|96| Changed|
|InteractionTag Converged.Outline.pressed content.chromium.png|30| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>